app/controllers/practices_controller.rb

```diff
@@ -466,3 +470,11 @@ def practices_json(practices)
     practices_array.to_json.html_safe
   end
 end
+
+def create_date_initiated (date_initiated)
```

Comment:
⚠️ Since we are manipulating the params, I felt it was best to put the logic in the controller as a private method.

Reply:
perfect